src/m_menu.c

```diff
@@ -2090,7 +2090,7 @@ static void M_DrawTabs(void)
         if (i == mult_screens_index)
         {
             V_FillRect(x + video.deltaw, rect->y + M_SPC, rect->w, 1,
-                       cr_gold[v_lightest_color]);
```

Comment:
Thanks! Have you checked if it makes a difference if you do `cr_shaded[(byte)cr_gold[v_lightest_color]]` or `cr_gold[(byte)cr_shaded[v_lightest_color]]`?

Reply:
For me there is not much difference.